cluster/addons/dns/coredns.yaml.base

```diff
@@ -74,23 +74,26 @@ metadata:
   name: coredns
   namespace: kube-system
   labels:
-    k8s-app: coredns
+    k8s-app: kube-dns
```

Comment:
I'm a bit confused on where the labels are being used as opposed to the Deployment name.

If I'm understanding this change and the cluster-proportional-autoscaler, this change is using the Deployment name to scale the dns component.

If there are tests that are using the wrong labels for verification, isn't that something that should be fixed up in the tests? Similar with kubeadm, if there are issues with the labels being set or queried with kubeadm, we should address those there.